app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -114,15 +114,15 @@ export const ContentForm: React.FC<IContentFormProps> = ({
     switch (contentType) {
       case ContentTypeName.PrintContent:
         return (a: IActionModel) =>
-          a.valueType === ValueType.Boolean && a.name !== ActionName.NonQualified;
+          a.isEnabled && a.valueType === ValueType.Boolean && a.name !== ActionName.NonQualified;
```

Comment:
Not an ideal solution.  Fix it at the source `ContentActions.tsx`

Reply:
Updated, thanks